src/trackingEvents.ts

```diff
@@ -138,6 +138,8 @@ export interface UserProfileChanged extends BaseEvent {
 type Primitive = null | string | number | boolean;
 type PrimitiveMap = {[member: string]: Primitive};
 type PrimitiveArray = Primitive[];
+type Map = {[member: string]: Primitive | PrimitiveMap | PrimitiveArray};
+type Array = PrimitiveArray | PrimitiveMap[] | PrimitiveArray[];
```

Comment:
Array is already a builtin type, should give other names here. Pehaps `TwoDimensionalMap` and `TwoDimensionalArray`?